### PR TITLE
Fixes issues due to serverspec upgrade 

### PR DESCRIPTION
**Pull request by cyberious**
and usage of obsolete should contain method for files

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -18,13 +18,18 @@ group :development, :unit_tests do
   gem 'simplecov',               :require => false
   gem 'puppet_facts',            :require => false
   gem 'json',                    :require => false
+  gem 'pry',                     :require => false
```
Comment (mhaskel) [inline Gemfile]: Did you intend to leave pry in here?

File: spec/acceptance/ini_setting_spec.rb [hunk 2]
```diff
@@ -1,19 +1,20 @@
 require 'spec_helper_acceptance'
+require 'pry'
```
Comment (mhaskel) [inline spec/acceptance/ini_setting_spec.rb]: Did you intend to leave pry in here?

Comment (cyberious) [inline spec/acceptance/ini_setting_spec.rb]: Not here no.

File: Gemfile (hunk 1 above)
Comment (cyberious) [inline Gemfile]: Yes

File: spec/acceptance/ini_setting_spec.rb [hunk 3]
```diff
@@ -23,20 +23,17 @@
 
     describe file(path) do
       it { should be_file }
-      #XXX Solaris 10 doesn't support multi-line grep
-      it("should contain #{content}", :unless => fact('osfamily') == 'Solaris') {
-        should contain(content)
-      }
+      its(:content) { should match content }
```
Comment (mhaskel) [inline spec/acceptance/ini_setting_spec.rb]: Did the serverspec update fix this on solaris 10?

Comment (cyberious) [inline spec/acceptance/ini_setting_spec.rb]: yes, it works, I have tested it against solaris 10, it was the grep that was broken, its(:content) does not use the multiline grep so with a regex it matches.

Comment (cyberious) [inline spec/acceptance/ini_setting_spec.rb]: should contain still uses the specinfra matcher, however the serverspec its(:content) would use the https://github.com/serverspec/serverspec/blob/5b7e90f259305726527253744f7a0feb3de1cae4/lib/serverspec/type/file.rb#L97-L102 and read as a string and then use rspec match regex to match the multiline string.

File: spec/acceptance/ini_setting_spec.rb [hunk 4]
```diff
@@ -72,13 +69,13 @@
         apply_manifest(pp, :catch_changes => true)
       end
 
-      it_behaves_like 'has_content', "#{tmpdir}/ini_setting.ini", pp, "four = five\n[one]\ntwo = three"
+      it_behaves_like 'has_content', "#{tmpdir}/ini_setting.ini", pp, /four = five\n\n\[one\]\ntwo = three/
```
Comment (mhaskel) [inline spec/acceptance/ini_setting_spec.rb]: why is there an extra `\n` added here?

Comment (cyberious) [inline spec/acceptance/ini_setting_spec.rb]: Every time I ran it with a regex it was reporting two returns.  Not sure if the grep was reporting differently or stdout, but it was consistent with all platforms on that failure.  Tested on CentOS 6, Win 2008/2012 and Solaris 10.

**Pull request merged**